infra/api.py

```diff
@@ -10,6 +10,8 @@
 
 NOPASSWORD = False
 
+QUIZ_TELEMETRY_FILE = 'quiz_telemetry.txt'
```

Comment:
Maybe end it with `.json`?

Reply:
It'll be a bunch of JSON objects, not suitable for parsing as-is. You'd have to wrap brackets around it. IDK if calling that `.json` will mess up any processing tools. I can call it that. Any other changes to that file name/path while I'm changing that?

Reply:
I think the file name is fine. JSON is sort of a disaster of a format, but I guess we can use `.json` anyway. As you can see earlier in the file, we use a single-object `.json` for some other data, but I agree that loading & saving the whole object isn't a good idea, it'll grow to eventually be super slow.